elastic/kibana
# [Infra UI] Normalise network and Disk rates for Hosts across time ranges

Closes #164152 

## Summary

This PR normalizes the disk and network charts per second. The charts changed: 
Disk IOPS (read/write) 
<img width="390" alt="diskiops" src="https://github.com/elastic/kibana/assets/14139027/d0993032-7309-4fb9-9e4e-0e9bce73e89e">
Network (rx/tx)
<img width="402" alt="network" src="https://github.com/elastic/kibana/assets/14139027/33b20570-60ca-48ef-839b-0363bcccd24a">
Disk Throughput (read/write)
<img width="590" alt="diskth" src="https://github.com/elastic/kibana/assets/14139027/115eba5a-7d46-4846-8f88-fe041b4cfeed">

## Testing

1. Go to hosts view and check network and disk charts (they should be normalized per second)
2. Open the host flyout and check network and disk charts (they should be normalized per second)


https://github.com/elastic/kibana/assets/14139027/087cd044-b6cc-4612-8fca-391a96848365
